src/express/upload.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const multer = require(`multer`);
+const path = require(`path`);
+const {nanoid} = require(`nanoid`);
+
+const UPLOAD_DIR = `../upload/img/`;
+const FILE_TYPES = [`image/png`, `image/jpg`, `image/jpeg`];
+const uploadDirAbsolute = path.resolve(__dirname, UPLOAD_DIR);
+
+const storage = multer.diskStorage({
+  destination: uploadDirAbsolute,
+  filename: (req, file, cb) => {
+    const uniqueName = nanoid(10);
+    const extension = file.originalname.split(`.`).pop();
+    cb(null, `${uniqueName}.${extension}`);
+  }
+});
+
+const fileFilter = (req, file, cb) => {
+  if (FILE_TYPES.includes(file.mimetype)) {
+    cb(null, true);
+  } else {
+    cb(null, false);
+  }
```

Comment:
```suggestion
    cb(null, FILE_TYPES.includes(file.mimetype));
```
Так будет то же самое